Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -74,6 +102,35 @@ def method_called?(node, method_name)
           false
         end
 
+        def method_called?(node, method_name)
+          # Check if method_name is called among the direct children nodes in the given node
+          node.each_child_node(:send) do |call_node|
+            next unless call_node.method_name == method_name
+            @offensive_node = call_node
+            @offense_source_range = call_node.source_range
+            return true
+          end
+          false
+        end
+
+        def check_precedence(first_nodes, next_nodes)
+          # Checks for precedence, returns the first pair of precedence violating nodes
+          next_nodes.each do |each_next_node|
+            first_nodes.each do |each_first_node|
+              return [each_first_node, each_next_node] if component_precedes?(each_first_node, each_next_node)
```

Comment:
Generally lines over 80 characters (or thereabouts) are to be avoided when possible.